front/app/containers/Admin/pages-menu/containers/BottomInfoForm/messages.ts

```diff
@@ -0,0 +1,21 @@
+import { defineMessages } from 'react-intl';
+
+export default defineMessages({
+  bottomInfoPageTitle: {
+    id: 'app.containers.Admin.pagesAndMenu.bottomInfoPageTitle',
+    defaultMessage: 'Bottom Info Section',
+  },
+  bottomInfoContentEditorTitle: {
+    id: 'app.containers.Admin.pagesAndMenu.bottomInfoContentEditorTitle',
+    defaultMessage: 'Content',
+  },
+  bottomInfoDescription: {
+    id: 'app.containers.Admin.pagesAndMenu.bottomInfoDescription',
+    defaultMessage:
+      'This empty section at the bottom of the homepage can be customized with your own images, text and links.',
+  },
+  bottomInfoSaveButton: {
+    id: 'app.containers.Admin.pagesAndMenu.bottomInfoSaveButton',
+    defaultMessage: 'Save Bottom Info Section',
+  },
+});
```

Comment:
Could you already fix the capitalization so we won't forget/we reduce work for Ben? To have them show up, delete their lines in en.json as well.